### ISSUE TITLE
Commissioning artwork - Can it handle extreme zoom?

### ISSUE THREAD

**Issue by erlend-sh** [fabricjs/fabric.js#1377]
Hi,

I'd like to commission a developer to help me create a simple mathematical art piece on behalf of the math games company [WeWantToKnow](http://wewanttoknow.com/).

The image will start off with some intersecting lines forming a shape. This is essentially an image zoomed in 1000% or more, and the user can scroll to zoom out until the full image fits the width of the screen. Naturally, an actual image of this size would be huge, so I'm thinking it would be better to draw it programmatically, which might also enable the line thickness to scale up a bit as you zoom out, so that they're not nearly invisible when zoomed all the way out.

Example:

![trianglezoom-close](https://cloud.githubusercontent.com/assets/583842/3189294/71f1f5e4-ecc2-11e3-824e-2ae7ac82851f.jpg)
![trianglezoom-far](https://cloud.githubusercontent.com/assets/583842/3189295/73e4d1fa-ecc2-11e3-8a5b-d383857f37e1.jpg)

Basically the reverse of zoom.it
http://zoom.it/

Could this be accomplished with fabric.js? If so, I'd love to get in touch with a freelancer. If not, I would greatly appreciate it if anyone could point me to a library/developer that's better suited to what I'm after.

Sincerely,
Erlend Sogge Heggen
WeWantToKnow
